sway/desktop/output.c

```diff
@@ -199,6 +199,14 @@ void output_layer_for_each_toplevel_surface(struct sway_output *output,
 	wl_list_for_each(layer_surface, layer_surfaces, link) {
 		struct wlr_layer_surface_v1 *wlr_layer_surface_v1 =
 			layer_surface->layer_surface;
+		struct fx_render_data *data = user_data;
+		struct layer_effects *effects = layer_surface->effects;
+		if (effects) {
+			data->deco_data.blur = effects->blur && config->blur_enabled;
+			data->deco_data.shadow = effects->shadow && config->shadow_enabled;
+			data->deco_data.corner_radius = effects->corner_rounding? config->corner_radius: 0;
```

Comment:
I think we can discuss omitting the `&& config->EFFECT` checks here. Seems weird to have a bar not be blurred if its specified in the layer config but disabled globally.